integrations/acquisition/covidcast/test_db_connection.py

```diff
@@ -0,0 +1,70 @@
+"""Unit tests for database.py."""
+
+# standard library
+import unittest
+from unittest.mock import MagicMock
+
+import mysql.connector
+from delphi.epidata.acquisition.covidcast.database import Database
+
+# py3tester coverage target
+__test_target__ = 'delphi.epidata.acquisition.covidcast.database'
+
+
+class UnitTests(unittest.TestCase):
+  """Basic unit tests."""
+
+  def test_connect_opens_connection(self):
+    """Connect to the database."""
+
+    database = Database()
+    database.connect(connector_impl=mysql.connector)
+    tmp_table_name = 'tmp_db_integration_test'
+    create_tmp_table_sql = f'''
+      CREATE TABLE `{tmp_table_name}` (
+        `source` varchar(32) NOT NULL,
+        `signal` varchar(64) NOT NULL,
+        `time_type` varchar(12) NOT NULL,
+        `geo_type` varchar(12) NOT NULL,
+        `time_value` int(11) NOT NULL,
+        `geo_value` varchar(12) NOT NULL,
+        `value_updated_timestamp` int(11) NOT NULL,
+        `value` double,
+        `stderr` double,
+        `sample_size` double,
+        `direction_updated_timestamp` int(11) NOT NULL,
+        `direction` int(11),
+        `issue` int(11) NOT NULL,
+        `lag` int(11) NOT NULL,
+        `is_latest_issue` BINARY(1) NOT NULL,
+        `is_wip` BINARY(1) NOT NULL,
+        `missing_value` int(1) DEFAULT 0,
+        `missing_stderr` int(1) DEFAULT 0,
+        `missing_sample_size` int(1) DEFAULT 0
+      ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+    '''
+    truncate_tmp_table_sql = f'TRUNCATE TABLE {tmp_table_name};'
+    drop_tmp_table_sql = f'DROP TABLE {tmp_table_name}'
+    breakpoint()
```

Comment:
what is this for?

Reply:
Sorry, I'm going to put this PR into a draft state, as I'm making some new changes to it. This file is my own db integration test before I found that the db setup/teardown code exists, my plan is to absorb it into existing tests.